video/out/wayland_common.c

```diff
@@ -2097,6 +2103,16 @@ static bool create_input(struct vo_wayland_state *wl)
     return 0;
 }
 
+static void xdg_activate(struct vo_wayland_state *wl)
+{
+    const char *token = getenv("XDG_ACTIVATION_TOKEN");
+    if (token) {
+        unsetenv("XDG_ACTIVATION_TOKEN");
```

Comment:
This call may invalidate `token` pointer or make it point to something else entirely. Move the `unsetenv` variable after all uses of the `token`.

> The getenv() function returns a pointer to the value in the environment

> The string pointed to by the return value of getenv() may be statically allocated, and can be modified by a subsequent call to getenv(), putenv(3), setenv(3), or unsetenv(3).